config/replaceobj.go

```diff
@@ -59,7 +59,7 @@ func jsonCompatTypes(val interface{}) error {
 				return err
 			}
 		}
-	case string, float64, bool, nil:
+	case string, float64, bool, int, nil:
```

Comment:
Not doing this is probably my only requirement here... everything else is nits, that you probably should address because it would be really easy :)